filebeat/harvester/reader.go

```diff
@@ -53,6 +55,7 @@ func newLogFileReader(
 		config:       config,
 		lastTimeRead: time.Now(),
 		backoff:      config.backoffDuration,
+		done:         done,
```

Comment:
@urso I tried this now in my new version and it actually leads to a race condition ...